process_cohort.smk

```diff
@@ -69,9 +69,9 @@ else:
 
 # scan smrtcells/ready directory for uBAMs or FASTQs that are ready to process
 # uBAMs have priority over FASTQs in downstream processes if both are available
-ubam_pattern = re.compile(r'smrtcells/ready/(?P<sample>[A-Za-z0-9_-]+)/(?P<movie>m\d{5}[Ue]?_\d{6}_\d{6}).(ccs|hifi_reads).bam')
+ubam_pattern = re.compile(r'smrtcells/ready/(?P<sample>[A-Za-z0-9_-]+)/(?P<movie>[A-Za-z0-9._-]+).bam')
```

Comment:
This is a very permissive regex that will potentially match aBAMs.

Reply:
i think it will be the responsibility of the user to keep abams out of smrtcells/ready/